### ISSUE TITLE
Doesn't work with Appends

### ISSUE THREAD

**Issue by GFWagnitz** [AuthentikCanada/eloquent-cache#3]
If a model has appends, not present on the database  but calculated, when the model is retrieved by cache and you try to save or update, it doesn't consider as an appended attribute. So it fails for the lack of the column on the database. 

E.g.:

Order Model doen't have price on the database, it calculates on load based on the products and the cupon used:

```
class OrderModel extends Model
{

    use SoftDeletes, Cacheable;

    protected $table = 'orders';

    protected $primaryKey = 'uuid';

    protected $keyType = 'string';

    public $incrementing = false;

    protected $fillable = [
        'user_uid',
        'cupon_uuid',
        'event_id',
        'order_items',
        'status'
    ];

    protected $casts = [
        'order_items' => 'array'
    ];

    protected $appends = [
        'full_price',
        'final_price'
    ];

    public function getFullPriceAttribute()
    {
        $full_price = 0;
        foreach($this->order_items as $item){
            $full_price =  $full_price + ItemModel::find($item['item_id'])->price;
        }

        return $this->attributes['full_price'] = $full_price;
    }

public function getFinalPriceAttribute()
    {
        $final_price = $this->full_price;

        if($this->cupon_uuid != null){
            $cupon = $this->cupon;
            $final_price = $final_price * (1 - ($cupon->value / 100));
            }
        }

        return $this->attributes['final_price'] = $final_price
    }

```

`Illuminate/Database/QueryException with message 'SQLSTATE[42703]: Undefined column: 7 ERROR:  column "full_price" of relation "orders" does not exist
LINE 1: ..._uid" = $6, "created_at" = $7, "deleted_at" = $8, "full_pric...`

**Comment by rundef** [comment]
Hi,

Thanks for reporting this bug. I'll look into it in the next couple of days.

**Comment by rundef** [comment]
Hello @GFWagnitz ,

This has been fixed in **1.1.5**.
You will have to flush the cache for your models that contain appends => `OrderModel::flush()`
